tests/test_genes.py

```diff
@@ -23,7 +23,7 @@
 
 CANONICAL_ONCOGENES = ["kras", "nras", "alk"]
 CANONICAL_TS = ["cdkn2a", "tp53"]
-CANONICAL_CG = ["ercc1", "fanci", "h2bc4", "h2bc17", "acvr1b"]
```

Comment:
just curious, were these genes not good tests?

Reply:
I first chose these gene randomly among the 500+ I had on hand, then I replaced them with gene 'alb' that I knew I had a related Statement in GKB. The list is here to be fill with more 'canonical' ones once we have some specific use cases we want to test for. I didn't wanted to make it looks as these genes were 'special', so I removed them.